@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -183,8 +198,19 @@ export class SimpleTileGeometryLoader implements TileGeometryLoader {
         enabledKinds: GeometryKindSet | undefined,
         disabledKinds: GeometryKindSet | undefined
     ): void {
-        if (this.m_decodedTile === undefined && this.m_tile.decodedTile !== undefined) {
-            this.setDecodedTile(this.m_tile.decodedTile);
+        const tile = this.tile;
+        let decodedTile = this.m_decodedTile;
+
+        // First time this tile is handled:
+        if (decodedTile === undefined && tile.decodedTile !== undefined) {
+            decodedTile = this.setDecodedTile(tile.decodedTile);
+            TileGeometryCreator.instance.processTechniques(
+                decodedTile,
+                enabledKinds,
+                disabledKinds
+            );
+
+            this.setDecodedTile(decodedTile);
```

Comment:
Is it necessary to do setDecodedTile twice? Can't we just processTechniques on tile.decodedTile, and set it afterwards?

Reply:
Missed that, fixed it.